frontend/web/src/styles/app.css

```
@@ -252,15 +252,18 @@ body {
-  width: 1px;
-  height: 1px;
+  position: static;
+  width: auto;
+  height: auto;
   padding: 0;
-  margin: -1px;
-  overflow: hidden;
-  clip: rect(0, 0, 0, 0);
+  margin: 0 0.85rem 0 0;
+  overflow: visible;
+  clip: auto;
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)